From: Phil Elwell <phil@raspberrypi.com>
Date: Mon, 18 May 2020 09:46:48 +0100
Subject: overlays: Fix audio parameter of vc4-kms-v3d

The CMA handling change broke the audio parameter - the fragment
numbering has changed - so fix it.

See: https://github.com/raspberrypi/linux/issues/2489

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
index b40aa1d8f432..caa64bfb95a7 100644
--- a/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
+++ b/arch/arm/boot/dts/overlays/vc4-kms-v3d-overlay.dts
@@ -109,6 +109,6 @@
 	};
 
 	__overrides__ {
-		audio   = <0>,"!17";
+		audio   = <0>,"!13";
 	};
 };
